pkg/process/net/windows_pipe.go

```diff
@@ -17,10 +28,29 @@ type WindowsPipeListener struct {
 	pipePath string
 }
 
-// NewListener sets up a TCP listener for now, will eventually be a named pipe
-func NewListener(socketAddr string) (*WindowsPipeListener, error) {
-	l, err := net.Listen("tcp", socketAddr)
-	return &WindowsPipeListener{l, "path"}, err
+// Create a standardized named pipe server and with hardened ACL
+func newPipeListener(namedPipeName string) (net.Listener, error) {
+	config := winio.PipeConfig{
+		InputBufferSize:  namedPipeInputBufferSize,
+		OutputBufferSize: namedPipeOutputBufferSize,
+	}
+
+	// TODO: Apply hardened ACL
```

Comment:
Should this ACL be part of the changes here?

Reply:
Not for this PR.  This is on-part with the previous socket-based endpoint (port 3333), since it was an open port.